bundles/binding/org.openhab.binding.weather/src/main/java/org/openhab/binding/weather/internal/model/Wind.java

```diff
@@ -22,19 +22,25 @@
  */
 public class Wind {
 
-    @ProviderMappings({ @Provider(name = ProviderName.WUNDERGROUND, property = "current_observation.wind_kph"),
+    @ProviderMappings({
+            @Provider(name = ProviderName.WUNDERGROUND, property = "current_observation.wind_kph"),
             @Provider(name = ProviderName.WUNDERGROUND, property = "avewind.kph"),
             @Provider(name = ProviderName.OPENWEATHERMAP, property = "speed"),
             @Provider(name = ProviderName.FORECASTIO, property = "windSpeed", converter = ConverterType.WIND_MPS),
             @Provider(name = ProviderName.WORLDWEATHERONLINE, property = "windspeedKmph"),
             @Provider(name = ProviderName.YAHOO, property = "wind.speed"),
-            @Provider(name = ProviderName.HAMWEATHER, property = "windSpeedKPH") })
+            @Provider(name = ProviderName.HAMWEATHER, property = "windSpeedKPH"),
+            @Provider(name = ProviderName.METEOBLUE, property = "wind_speed"),
+            @Provider(name = ProviderName.METEOBLUE, property = "wind_speed_max") })
```

Comment:
Do `wind_speed` and `wind_speed_max` mean the same thing?  It seems (without research) that they should be two distinct things: the current sample of wind speed, and then a max wind gust speed over some previous period.  For example, "current wind speed 20 km/h, gusting to 45 km/h."